framework/src/transfers/MultiAppVariableValueSamplePostprocessorTransfer.C

```diff
@@ -41,15 +47,104 @@ MultiAppVariableValueSamplePostprocessorTransfer::validParams()
 MultiAppVariableValueSamplePostprocessorTransfer::MultiAppVariableValueSamplePostprocessorTransfer(
     const InputParameters & parameters)
   : MultiAppTransfer(parameters),
+    MeshChangedInterface(parameters),
     _postprocessor_name(getParam<PostprocessorName>("postprocessor")),
-    _from_var_name(getParam<VariableName>("source_variable")),
-    _comp(getParam<unsigned int>("source_variable_component"))
+    _var_name(getParam<VariableName>("source_variable")),
+    _comp(getParam<unsigned int>("source_variable_component")),
+    _var(_fe_problem.getVariable(0, _var_name))
 {
   if (_directions.size() != 1)
     paramError("direction", "This transfer is only unidirectional");
 
-  if (isParamValid("from_multi_app"))
-    paramError("from_multi_app", "This transfer direction has not been implemented");
```

Comment:
we still need an error for the sibling case since it was not implemented?

Reply:
Is the above check `if (_directions.size() != 1)` covering this? I assume you mean the transfers among siblings.

Reply:
sibling would be one direction still, between_multiapp